### PR TITLE
Supports manually reset and OOP API.

### PR DESCRIPTION
**Pull request by zeriyoshi**
Add OOP API with Manually zero-point reset function.

Old API still supported, but doesn't have zero-point reset function.

### CODE REVIEW

File: cmd/mzh19b/main.go [hunk 1]
```diff
@@ -3,13 +3,21 @@ package main
...
 func main() {
+	calibrate := flag.Bool("r", false, "before calibrate MH-Z19B")
 	device := flag.String("device", "/dev/serial0", "specific MH-Z19B")
 	flag.Parse()
 
-	ppm, _ := mhz19b.Read(*device)
+	mhz19b := mhz19b.New(*device)
+
+	if *calibrate {
+		mhz19b.CalibrateDefault()
```
Comment (macaron) [inline cmd/mzh19b/main.go]:
```suggestion
		_ = mhz19b.CalibrateDefault()
```

File: README.md [hunk 2]
```diff
@@ -16,11 +16,15 @@ $ go get -u github.com/macaron/go-mh-z19b
 
 ```go
 func main() {
-    ppm, err := mhz19b.Read("/dev/serial0")
+    mhz16b := mhz19b.New("/dev/serial0")
```
Comment (macaron) [inline README.md]: What is mhz16b?

```suggestion
    mhz19b := mhz19b.New("/dev/serial0")
```

File: README.md [hunk 3]
```diff
@@ -16,11 +16,15 @@ $ go get -u github.com/macaron/go-mh-z19b
...
+    ppm, err := mhz19b.Read()
     
     if err != nil {
     	log.Fetal(err)
     }
     fmt.Printf("CO2 = %dppm\n", ppm)
+
+    // If you want to reset the sensor ZERO point
+    // mhz19b.CalibrateDefault()
```
Comment (macaron) [inline README.md:28]: 👍

File: cmd/mzh19b/main.go [hunk 4]
```diff
@@ -3,13 +3,21 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/macaron/go-mh-z19b"
+
+	mhz19b "github.com/macaron/go-mh-z19b"
```
Comment (macaron) [inline cmd/mzh19b/main.go:7]: Why add a alias?
In many cases, Use this alias when import different packages of same name.

File: cmd/mzh19b/main.go [hunk 5]
```diff
@@ -3,13 +3,21 @@ package main
...
 )
 
 func main() {
+	calibrate := flag.Bool("r", false, "before calibrate MH-Z19B")
 	device := flag.String("device", "/dev/serial0", "specific MH-Z19B")
 	flag.Parse()
 
-	ppm, _ := mhz19b.Read(*device)
+	mhz19b := mhz19b.New(*device)
```
Comment (macaron) [inline cmd/mzh19b/main.go]: GoLand suggested "Variable 'mhz19b' collides with imported package name."
Uh...

golang user like very short variables.
```suggestion
	m := mhz19b.New(*device)
```

or

```suggestion
	co2 := mhz19b.New(*device)
```

**Review by macaron — CHANGES_REQUESTED** [review]
Nice contribute.Just a little bit more!
I wrote nitpick comments.
Thank you.

File: cmd/mzh19b/main.go (hunk 4 above)
Comment (zeriyoshi) [reply cmd/mzh19b/main.go:7]: This was automatically completed by VSCode's Go plugin

File: cmd/mzh19b/main.go [hunk 6]
```diff
@@ -3,13 +3,21 @@ package main
...
 func main() {
+	calibrate := flag.Bool("r", false, "before calibrate MH-Z19B")
 	device := flag.String("device", "/dev/serial0", "specific MH-Z19B")
 	flag.Parse()
 
-	ppm, _ := mhz19b.Read(*device)
+	m := mhz19b.New(*device)
+
+	if *calibrate {
+		_ = mhz19b.CalibrateDefault()
```
Comment (macaron) [inline cmd/mzh19b/main.go]:
```suggestion
		_ = m.CalibrateDefault()
```

File: cmd/mzh19b/main.go [hunk 7]
```diff
@@ -3,13 +3,21 @@ package main
...
+	}
+
+	ppm, _ := mhz19b.Read()
```
Comment (macaron) [inline cmd/mzh19b/main.go]:
```suggestion
	ppm, _ := m.Read()
```

**Review by macaron — CHANGES_REQUESTED** [review]
Please fix variables 🙏

**Pull request merged**